FlowCryptUITests/SignInImapTest.swift

```diff
@@ -0,0 +1,308 @@
+//
+//  SignInViewControllerTest.swift
+//  FlowCryptUITests
+//
+//  Created by Anton Kharchevskyi on 18.10.2019.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+import FlowCryptCommon
+
+// MARK: - Compatibility account
+
+class SignInImapTest: XCTestCase, AppTest {
+    var app: XCUIApplication!
+
+    override func setUp() {
+        continueAfterFailure = false
+
+        logger.logInfo("Start App")
+        
+        app = XCUIApplicationBuilder()
+            .setupRegion()
+            .build()
+            .addSnapshot()
+            .launched()
+        
+        logger.logInfo("Wait for launch")
+        wait(10)
+    }
+}
+
+extension SignInImapTest {
+    // login -> approve -> backups found -> enter pass phrase -> main flow
+    func test_1_successful_login_imap() {
+        let user = UserCredentials.imapDev
+        login(user)
+        
+        
+        passPhraseTextField.tap()
+        passPhraseTextField.typeText(user.pass)
+        goKeyboardButton.tap()
+        
+        wait(4)
+        XCTAssert(app.buttons["+"].exists)
+    }
+}
+
+//extension SignInImapTest {
```

Comment:
Commented out existed tests. Need to readable them in separate ticket/tickets

Reply:
Got it, I filed an issue separately.